### PR TITLE
fix: calculate griefing collateral based on input amount

### PR DESCRIPTION
**Pull request by gregdhill**
Signed-off-by: Gregory Hill <gregorydhill@outlook.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,13 +23,14 @@
     "test": "run-s build test:*",
     "test:lint": "eslint src --ext .ts",
     "test:unit": "mocha test/unit/*.test.ts test/unit/**/*.test.ts",
-    "test:integration": "mocha test/integration/**/*.test.ts --timeout 1000000",
+    "test:integration": "mocha test/integration/**/issue.test.ts --timeout 1000000",
```
Comment (savudani8) [inline package.json:26]: Please revert this change

Comment (gregdhill) [reply package.json:26]: Oops

**Review by savudani8 — APPROVED** [review]
